keras/src/backend/tensorflow/image.py

```diff
@@ -15,6 +15,19 @@
 )
 
 
+def psnr(image1, image2, max_val):
+    from keras.src.backend.tensorflow.numpy import log10
+    if image1.shape != image2.shape:
+        raise ValueError(
+            f"Image shapes {image1.shape} and {image2.shape} must match for PSNR calculation."
```

Comment:
Reproduce the same error message as in the other functions.

Reply:
Ok